OKR/PROJECT.md

```diff
@@ -2,8 +2,29 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
 
+- **The IPFS Org gets a fully dedicated Package Managers Working Group**
+  - `P0` - @daviddias - A Package Manager Working Group Roadmap is created
+  - `P1` - @??? - Onboard 1+ contributors to Package Managers Working Group
+- **Support and guide the IPFSConf planning and Content Curation**
+  - `P0` - @daviddias - The IPFSConf is scheduled, announced and there is a CFP running. The IPFSConf Org Team is supported by the IPFS org.
+  - `P0` - @momack2 - The IPFS Org (Working Groups & Core Contributors) has a unified plan with goals for IPFSConf
+- **The IPFS Working Groups feel supported and they have the information they need to make the best decisions**
+  - `P0` - @mikeal - We gather, process and present metrics for the IPFS Project  
+  - `P1` - @parkan - Ship the first IPFS User Quarterly Report. It documents achievements and demos of new capabilities
+  - `P1` - @momack2 - Working group roadmaps are solidified, presented, and used to chart Q1 and Q2 planning
+- **The IPFS Community is supportive, responsive, and easy to get onboarded to**
+  - `P0` - @parkan - There are zero dropped/mishandled collabs  
+  - `P1` - @parkan - Biweekly updates published to IPFS working groups highlighting patterns in top developer needs
+  - `P1` - @daviddias - The Working Groups are listed on the IPFS Website (ref: ipfs/project#14)
+  - `P2` - @parkan - Ship strategy document outlining tooling, pipeline, and management for collabs with the IPFS Project  
+  - `P2` - @parkan - 10 collab engagements result in valuable output for the wider community (feature, blog post, collaborative meeting, bug fix, etc)  
+- **The IPFS Projects have assigned maintainers, are well documented and users have good resources to rely on**
+  - `P0` - @daviddias - Reboot the IPFS Specifications repo and create a coordinated effort to write and maintain the specifications
+  - `P1` - @daviddias - Expand the Lead Maintainer Protocol to become the Maintainer Protocol for the entire organization
+  
```

Comment:
```suggestion
- `P1` - @meiqimichelle - The IPFS Project onboards at least +1 new UX Research contributor to help us make more user-centered decisions
```
This is one of our milestones on our Q1 roadmap - does it deserve a mention here?

Reply:
Through offline convo I think this is good =]